packages/payment-processor/src/payment/utils.ts

```diff
@@ -306,6 +313,65 @@ export function validateConversionFeeProxyRequest(
   }
 }
 
+/**
+ * Validates the parameters for an ERC20 Fee Proxy payment before payment
```

Comment:
Header comment could be more precise.

```suggestion
 * Validates the parameters for an ERC20 Transferable Receivable payment, esp. that token exists 
```